src/components/Testimonials/Testimonials.js

```diff
@@ -0,0 +1,33 @@
+import './Testimonials.scss';
```

Comment:
A `Testimonials` React component already exists in `src/views`, which is also where a component like this belongs. However, I don't think this needs to be its own component anyways since it only renders a few paragraphs. Instead, its logic should just be moved to `OurImpactPage.js`.

Also, please take a look at the README again to clear up the semantic difference between `components` and `views`. They're both ultimately React elements, but differ slightly in their purpose/reusability.